### PR TITLE
Inbound Transfer Integration

### PR DESCRIPTION
**Pull request by truthfool**
## Description

- added response object necessary fields

 _(Ignore if these details are present on the associated JIRA ticket)_

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Design related bullet points or design document link related to this PR added in the description above. 

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

**Comment by truthfool** [comment]
@fynmanoj @avikganguly01

File: src/main/java/org/mifos/connector/ams/pesacore/camel/route/PesaRouteBuilder.java [hunk 1]
```diff
@@ -62,7 +62,8 @@ public void configure() {
                 .setBody(e -> {
                     String body=e.getIn().getBody(String.class);
                     logger.debug("Body : {}",body);
-                    e.setProperty("dfspId",e.getProperty("dfspId"));
+                    String accountHoldingInstitutionId= String.valueOf(e.getIn().getHeader("accountHoldingInstitutionId"));
```
Comment (fynmanoj) [inline src/main/java/org/mifos/connector/ams/pesacore/camel/route/PesaRouteBuilder.java:65]: why do we need accountHoldingInstitutionId in pesa?

**Review by fynmanoj — CHANGES_REQUESTED** [review]
clarification : why do we need accountHoldingInstitutionId in pesa?

File: src/main/java/org/mifos/connector/ams/pesacore/camel/route/PesaRouteBuilder.java (hunk 1 above)
Comment (truthfool) [reply src/main/java/org/mifos/connector/ams/pesacore/camel/route/PesaRouteBuilder.java:65]: This gets passed from response back to mpesa->channel to start the workflow.

**Pull request merged**